core/node/api_server/src/web3/namespaces/eth.rs

```diff
@@ -689,6 +689,8 @@ impl EthNamespace {
         effective_pubdata_price_history.reverse();
 
         let oldest_block = newest_l2_block.0 + 1 - base_fee_per_gas.len() as u32;
+        // `base_fee_per_gas` for next L2 block cannot be calculated, appending last fee as a placeholder.
```

Comment:
Drive-by off-topic (?) question: Why can't it be calculated? Isn't this only the case if `newest_l2_block` is the latest one? Otherwise, the next block info can be requested from the DB.

Reply:
Aha, so because such calculation is impossible, we prefer to return a dummy value for the next block instead of loading it from the DB? IIUC, Ethereum Web3 providers return the next block value when queried non-latest blocks, but they *are* calculated per EIP1559.

Reply:
I don't have a strong opinion on this. Anyway users should not rely on `next_base_fee`